services/rds/drivers/RdsCommon.py

```diff
@@ -352,7 +352,7 @@ def _checkIfAurora(self):
             self.results['ConsiderAurora'] = [-1, self.engine]
     
     def _checkHasGravitonOption(self):
-        if self.isCluster == True:
+        if self.isCluster == True or self.isServerless == True:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `==` and `!=` operators use the compared objects&#39; ` __eq__` method to test if they are equal. To check if an object is a singleton, such as `None`, we recommend that you use the `is` identity comparison operator.

[Learn more](https://docs.python.org/3/reference/expressions.html#is)